src/main/scala/example/BookBot.scala

```diff
@@ -0,0 +1,32 @@
+package example
+
+import scalaj.http._
+import io.circe._, io.circe.parser._
+
+object BookBot extends App {
+  def parseJson(json: String): Json = {
+    parse(json).getOrElse(Json.Null)
```

Comment:
`getOrElse` 😎 👍

Reply:
Another thing you could look into is using a `case` statement when calling a function that returns an `either`: 
```
 parseJson(json) match {
      case Left(s) => doAThing
      case Right(i) => doAnotherThing
  }
```